sourmash/cli/index.py

```diff
@@ -44,6 +44,13 @@ def subparser(subparsers):
         '--scaled', metavar='FLOAT', type=float, default=0,
         help='downsample signatures to the specified scaled factor'
     )
+    subparser.add_argument(
+        '--not-localized', action='store_true', default=False,
+        help='Do not create a localized SBT index. A localized index guarantees that '
+             'any two leaves sharing a parent are more similar to leaves not sharing '
+             'a parent. Localized indices are required for building a nearest neighbor'
+             ' graph from the SBT.'
+    )
```

Comment:
I don't think this option is necessary: if `d != 2` then build the current `SBT`, otherwise always build the `LocalizedSBT`.

Reply:
well, it's nice for debugging :)